code/__HELPERS/icons.dm

```diff
@@ -906,3 +906,39 @@ proc/get_icon_difference(var/icon/main, var/icon/mask)
 		mask.BecomeAlphaMask() //Make all the black pixels vanish (fully transparent), leaving only the white pixels.
 		main.AddAlphaMask(mask) //Make the pixels in the main icon that are in the transparent zone of the mask icon also vanish (fully transparent).
 		return main
+
+//For creating consistent icons for human looking simple animals
+/proc/get_flat_human_icon(icon_id, datum/job/J, datum/preferences/prefs, dummy_key)

```

Comment:
/tg/ has since updated this and added more features, I would suggest porting as much as you can until it stops working. I think compile_overlays isn't necessary, but if it is, nevermind really.